src/Project.js

```diff
@@ -81,7 +84,51 @@ export default class Project {
     this.input.focus();
   }
 
+  // Find triggers which match the given condition
+  _matchingTriggers(triggerMatches) {
+    let matchingTriggers = [];
+    const targets = this.spritesAndStage;
+    for (const target of targets) {
+      const matchingTargetTriggers = target.triggers.filter(tr =>
+        triggerMatches(tr, target)
+      );
+      for (const match of matchingTargetTriggers) {
+        matchingTriggers.push({ trigger: match, target });
+      }
+    }
+    return matchingTriggers;
+  }
+
+  _stepEdgeActivatedTriggers() {
+    const edgeActivated = this._matchingTriggers(tr => tr.isEdgeActivated);
+    const triggersToStart = [];
+    for (const triggerWithTarget of edgeActivated) {
+      const { trigger } = triggerWithTarget;
+      let predicate;
+      switch (trigger.trigger) {
+        case Trigger.TIMER_GREATER_THAN:
+          predicate = this.timer > trigger.option("VALUE");
```

Comment:
You added support for function options which can receive the `target` as an argument, but then you are not passing `target` into `trigger.option()` here. Is there a reason why you don't pass it?

Reply:
Nope, just an oversight